data/transport missions.txt

```diff
@@ -1412,6 +1415,138 @@ mission "Hallucination"
+
+
+
+mission "Shady passenger transport 2"
+	name "Shady passenger transport"
+	description "Discreetly transport two passengers to <destination>. If you are caught with them, you may be fined."
+	minor
+	repeat
+	source
+		attributes "near earth" core urban
+	destination
+		attributes "frontier" "dirt belt" "rim" "north" "south" "pirate"
+		distance 3 7
+	passengers 2
+	illegal 150000 `Your passengers turn out to be wanted criminals. As they are hauled off in chains, you vehemently protest that you knew nothing of their shady histories. The authorities threaten you with jail time, and you quickly contact a lawyer. You wind up quite a bit poorer through a combination of fines, bribes, and attorney's fees (the breakdown remains somewhat unclear).`
+		stealth
+
+	to offer
+		random < 8
+		"rejected illegal jobs" < 3
+		"getaway driver" > 2
+
+	on offer
+		dialog `As you are walking through the spaceport, two men in nondescript clothing appear out of nowhere and offer <payment> for discreet, no-questions-asked passage to <destination>.`
+
+	on accept
+		"rejected illegal jobs" --
+		"getaway driver" ++
+		dialog `You quickly bundle the men onto your ship before anyone sees them. As you're helping unpack their luggage, one of them casually mentions that they may have attracted the attention of some bounty hunters, and would prefer not to be captured. You should probably avoid being scanned, too."`
+
+	on decline
+		"rejected illegal jobs" ++
+
+	on complete
+		"getaway driver" ++
+		payment 100000
+		dialog `Your passengers depart after paying you the agreed-upon sum of <payment>, off to make new lives for themselves doing who-knows-what.`
+
+	npc
+		government "Bounty Hunter"
+		personality nemesis plunders waiting
+		system
+			distance 6 9
+		fleet "Bounty Hunters"
+
+
+
+mission "Shady passenger transport 3"
+	name "Shady passenger transport"
+	description "Discreetly transport <bunks> passengers to <destination>. If you are caught with them, the legal consequences may be severe."
+	minor
+	repeat
+	source
+		attributes "frontier" "dirt belt" "rim" "north" "south" "pirate"
+	destination
+		attributes "near earth" "core" "urban"
+		distance 5 10
+	passengers 4 13
+	illegal 400000 `It's difficult to feign ignorance of your passengers' background during your questioning. You wind up having to pay a small fortune in bribes to keep yourself from being charged as an accomplice to some very serious-sounding crimes.`
+		stealth
+
+	to offer
+		random < 5
+		"rejected illegal jobs" < 3
+		"getaway driver" > 5
+		has "Shady passenger transport 2: done"
+
+	on offer
+		dialog `A small group of fearsome-looking young men and women brazenly walk up to you in broad daylight, explaining that they're in some legal trouble and need to get to <destination>, offering <payment> to secure discreet passage. You must be getting quite a reputation in certain circles. Transporting them might be very dangerous. On the other hand, they're offering a small fortune...`
```

Comment:
4 to 13? 13 doesn't sound like a very small group to me. Perhaps change this to `A group of <bunks>...` so the player knows some actual numbers.